content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -92,6 +92,18 @@ Common forms of values include:
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### kubectl.kubernetes.io/restartedAt
+
+Example: `kubectl.kubernetes.io/restartedAt: "2023-03-31T15:30:00Z"`
+
+Used on: Pod
```

Comment:
Per my understanding,  this annotation could be applied to DaemonSets, Deployments, StatefulSets too by kubectl when you execute the `kubectl rollout restart` command. However, I'll let a tech reviewer from SIG CLI confirm this.

Reply:
Tested this on a deployment, the Pod had the following annotation added:
`kubectl.kubernetes.io/restartedAt: '2023-04-04T14:07:20Z'`